onnxruntime/contrib_ops/cuda/grid_sample.cc

```diff
@@ -9,22 +9,23 @@ namespace onnxruntime {
 namespace contrib {
 namespace cuda {
 
-#define REGISTER_KERNEL_TYPED(T)                                   \
+#define REGISTER_KERNEL_TYPED(T, VERSION, LAYOUT, DOMAIN)                   \
   ONNX_OPERATOR_TYPED_KERNEL_EX(                                   \
       GridSample,                                                  \
-      kMSDomain,                                                   \
-      1,                                                           \
+      DOMAIN,                                                      \
+      VERSION,                                                          \
```

Comment:
nit: Format code with Clang-format like https://skillupwards.com/blog/formatting-c-cplusplus-code-using-clangformat-and-vscode.

Reply:
Done.